src/tests/ftest/io/nvme_io.py

```diff
@@ -24,81 +24,52 @@
 from __future__ import print_function
 
 import os
-import sys
-import json
 import traceback
 import uuid
 import re
 import avocado
+from apricot import TestWithServers
 
-sys.path.append('./util')
-sys.path.append('../util')
-sys.path.append('../../../utils/py')
-sys.path.append('./../../utils/py')
-import server_utils
 import write_host_file
 import ior_utils
-import AgentUtils
 import spdk
 
 from general_utils import DaosTestError
-from daos_api import DaosContext, DaosPool, DaosApiError, DaosLog
+from daos_api import DaosPool, DaosApiError
 
-class NvmeIo(avocado.Test):
+class NvmeIo(TestWithServers):
     """
-    Test Class Description:
-        Test the general Metadata operations and boundary conditions.
+    Run IO over NVMe and verify the pool size.
+    :avocado: recursive
     """
-
     def setUp(self):
         self.pool = None
-        self.hostlist = None
         self.hostfile_clients = None
```

Comment:
This should be defined in the apricot classes' __init__() already:
https://github.com/daos-stack/daos/blob/0562c1f9d8c5b842952883870850ec2b6fe0bc0a/src/tests/ftest/util/apricot/apricot/test.py#L81